### DISCUSSION TITLE
It seem like winget using that link that expired too frequently in imagemagick.org.

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [ImageMagick/ImageMagick#6832] [General] [answered]
That link is expired too frequently in imagemagick.org which causes to update manifest. Winget can download from github and sourceforge.net but there is no available installer. 
See https://github.com/microsoft/winget-pkgs/issues/120261#issuecomment-1784188875
Does any alternative way to download and the link is permanent

**Comment by dlemstra** [comment]
We might explore providing an an installer in the Windows Store and Winget. We are talking with Microsoft about this.

**Comment by otto-liljalaakso-nt** [comment, marked as answer]
How about keeping also older versions available at https://imagemagick.org/archive/binaries/? Or providing the binaries from GitHub releases? Either of those would solve the WinGet issue right now. Is it just a disk space issue (as I see you provide a very large amount of binaries)? Or is there some other reason for purging everything but the latest binaries?

**Reply by dlemstra** [reply]
We haven't yet decided how we want to resolve this so for now this will remain broken. We will probably want to add it to our GitHub releases but we will need to investigate that.
